SA3/qa_web/templates/qa_web/EditUserProfile.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Edit User Profile</title>
+    </br>
+    <h1> Edit User Profile</h1>
+
+</head>
+<body>
+
+<!–– Django supported form is looped below to obtain the information -->
+    <form method="post">
```

Comment:
Much better and more compact template for editing user profile.

Reply:
removed.